apps/src/constants.js

```diff
@@ -176,3 +176,6 @@ export const ALPHABET =
   'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
 export const CIPHER =
   'Iq61F8kiaUHPGcsY7DgX4yAu3LwtWhnCmeR5pVrJoKfQZMx0BSdlOjEv2TbN9z';
+
+export const EXPO_SESSION_SECRET =
+  '{"id":"450308ef-67ec-4314-a438-60589b9c0fa2","version":1,"expires_at":2000000000000}';
```

Comment:
NOTE: this is not a valid secret. It is here only for development purposes as a fallback so you can figure out where to swap in your own secret for testing.

Reply:
Can we make that more obvious? Put `fake-fake-fake` in the middle or something?

Reply:
What does the user experience look like here? I'm not sure I understand when/how the new key gets swapped in for the "real" export flow.

Reply:
It comes in via `appOptions` with chef - but not on local dev systems.